styles/nixos-manual/catppuccin.user.css

```
@@ -0,0 +1,200 @@
+    @yellow: @catppuccin[@@lookup][@yellow];
+    @green: @catppuccin[@@lookup][@green];
+    @teal: @catppuccin[@@lookup][@teal];
+    @sky: @catppuccin[@@lookup][@sky];
+    @sapphire: @catppuccin[@@lookup][@sapphire];
+    @blue: @catppuccin[@@lookup][@blue];
+    @lavender: @catppuccin[@@lookup][@lavender];
+    @text: @catppuccin[@@lookup][@text];
+    @subtext1: @catppuccin[@@lookup][@subtext1];
+    @subtext0: @catppuccin[@@lookup][@subtext0];
+    @overlay2: @catppuccin[@@lookup][@overlay2];
+    @overlay1: @catppuccin[@@lookup][@overlay1];
+    @overlay0: @catppuccin[@@lookup][@overlay0];
+    @surface2: @catppuccin[@@lookup][@surface2];
+    @surface1: @catppuccin[@@lookup][@surface1];
+    @surface0: @catppuccin[@@lookup][@surface0];
+    @base: @catppuccin[@@lookup][@base];
+    @mantle: @catppuccin[@@lookup][@mantle];
+    @crust: @catppuccin[@@lookup][@crust];
+    @accent-color: @catppuccin[@@lookup][@@accent];
+
+    --ctp-rosewater: @rosewater;
+    --ctp-flamingo: @flamingo;
+    --ctp-pink: @pink;
+    --ctp-mauve: @mauve;
+    --ctp-red: @red;
+    --ctp-maroon: @maroon;
+    --ctp-peach: @peach;
+    --ctp-yellow: @yellow;
+    --ctp-green: @green;
+    --ctp-teal: @teal;
+    --ctp-sky: @sky;
+    --ctp-sapphire: @sapphire;
+    --ctp-blue: @blue;
+    --ctp-lavender: @lavender;
+    --ctp-text: @text;
+    --ctp-subtext1: @subtext1;
+    --ctp-subtext0: @subtext0;
+    --ctp-overlay2: @overlay2;
+    --ctp-overlay1: @overlay1;
+    --ctp-overlay0: @overlay0;
+    --ctp-surface2: @surface2;
+    --ctp-surface1: @surface1;
+    --ctp-surface0: @surface0;
+    --ctp-base: @base;
+    --ctp-mantle: @mantle;
+    --ctp-crust: @crust;
+
+    color-scheme: if(@lookup = latte, light, dark);
+
+    ::selection {
+      background-color: fade(@accent-color, 30%);
+    }
+
+    body {
+      background-color: @base !important;
+      color: @text;
+    }
+
+    p,
+    ol,
+    ul {
+      color: @text !important;
+    }
+
+    a {
+      border-bottom-color: @accent-color !important;
+      color: @accent-color;
+    }
+
+    h1,
+    h2,
+    h3,
+    h4 {
+      color: @accent-color;
+    }
+    h5 {
+      color: fade(@accent-color, 85%);
+    }
+
+    hr,
+    div.toc {
+      border-top-color: transparent;
+      border-bottom-color: @overlay1 !important;
+    }
+
+    div.variablelist {
+      color: @text;
+    }
+
+    code {
+      &.programlisting.hljs {
+        background-color: fade(@blue, 5%) !important;
+      }
+    }
+
+    table.informaltable {
+      border-color: @surface1;
+      thead {
+        background-color: @mantle;
+      }
+    }
+
+    #callout(@color) {
+      background-color: fade(@color, 7%) !important;
+      border-color: fade(@color, 20%);
+
+      a,
+      p,
+      .title {
+        color: @color !important;
+      }
+      a {
+        border-bottom-color: @color !important;
+      }
+    }
+
+    .note {
+      #callout(@blue);
+    }
+
+    .tip {
+      #callout(@green);
+    }
+
```

See https://daiderd.com/nix-darwin/manual/index.html#opt-launchd.daemons._name_.serviceConfig.StartCalendarInterval (or cmd + f search for "important"), https://github.com/NixOS/nixpkgs/blob/5e4fbfb6b3de1aa2872b76d49fafc942626e2add/pkgs/by-name/ni/nixos-render-docs/src/nixos_render_docs/commonmark.py#L129-L132.

```suggestion

    .important {
      #callout(@mauve);
    }

```

---

nice catch